File: Feitos em Java/2548.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while(leitor.hasNext()){
            int n = leitor.nextInt();
            int m = leitor.nextInt();
            int resu=0;
            for(int i=0;i<n;i++){
                int input = leitor.nextInt();
                if(i>=n-m){
                    resu+=input;
                }
            }
            System.out.println(resu);
	}
    }
}

File: Feitos em C/2548.c
#include <stdio.h>
int main() {
	int n,m,i;
	while(scanf("%d",&n)==1){
		scanf("%d",&m);
		int input,resu=0;
		for(i=0;i<n;i++){
			scanf("%d",&input);
			if(i>=n-m){
				resu+=input;
			}
		}
		printf("%d\n",resu);
	}
    return 0;
}
